docs/about.rst

```diff
@@ -7,11 +7,11 @@ Precision medicine refers to the use of prevention and treatment strategies that
 
 .. rubric:: How our Collaborative Process is Different
 
-All content created in CIViC is, and will continue to be, freely available, without restriction with a `Public Domain License <https://creativecommons.org/publicdomain/zero/1.0/>`_. Anyone can contribute to CIViC by simply registering for an account. Users join the community with "Curator" level privileges that allow them to comment or propose additions or revisions on any knowledge in CIViC. Proposed additions and revisions are then reviewed by users with "Editor" level privileges before entering the public CIViC record. Community curators work together with editors towards comprehensive and accurate interpretations of the clinical relevance of variants in cancer.
+All content created in CIViC is, and will continue to be, freely available, without restriction with a `Public Domain License <https://creativecommons.org/publicdomain/zero/1.0/>`_. Anyone can contribute to CIViC by simply registering for an account. Users join the community with "Curator" level privileges that allow them to comment or propose additions or revisions on any knowledge in CIViC. Proposed additions and revisions are then reviewed by users with "Editor" level privileges before gaining "Accepted" status. Submitted content, not yet accepted, should be interpreted/used with caution. Community curators work together with editors towards comprehensive and accurate interpretations of the clinical relevance of variants in cancer.
```

Comment:
should `curators` and `editors` in the last sentence be capitalized?

Reply:
Sure

Reply:
changes made